### PR TITLE
Better OS support for init_style

### PR DESCRIPTION
**Pull request by avishai-ish-shalom**
Support non LTS ubuntu versions, etc.

### CODE REVIEW

File: spec/classes/init_spec.rb [hunk 1]
```diff
@@ -255,6 +257,7 @@
     let(:facts) {{
       :operatingsystem => 'Fedora',
       :operatingsystemmajrelease => 20,
```
Comment (solarkennedy) [inline spec/classes/init_spec.rb]: is operatingsystemmajrelease no longer used? if so can you remove it to prevent confusion?

Comment (avishai-ish-shalom) [inline spec/classes/init_spec.rb]: sure.

**Comment by avishai-ish-shalom** [comment]
fixed

**Comment by solarkennedy** [comment]
Thanks!

**Pull request merged**